src/components/content/order/formElements/SelectField.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { Form, Image, Select } from 'antd';
+import { DeployParam, SelectOnChangeHandler } from './CommonTypes';
+import { CloudServiceProvider, CredentialVariables } from '../../../../xpanse-api/generated';
+import { cspMap } from './CspSelect';
+
+export function SelectField({
+    item,
+    onChangeHandler,
+}: {
+    item: DeployParam;
+    onChangeHandler: SelectOnChangeHandler;
+}): React.JSX.Element {
+    const activeCspList: CredentialVariables.csp[] = [
```

Comment:
please remove this definition of eum here again. 
Also the selectField must handle any enum values. Not just Csp